plugins/modules/eos_route_maps.py

```diff
@@ -0,0 +1,1404 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2021 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+
+"""
+The module file for eos_route_maps
+"""
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+module: eos_route_maps
+short_description: Manages Route Maps resource module
+description: This module configures and manages the attributes of Route Mapd on Arista
+  EOS platforms.
+version_added: 2.1.0
+author: Gomathi Selvi Srinivasan (@GomathiselviS)
+notes:
+- Tested against Arista EOS 4.23.0F
+- This module works with connection C(network_cli). See the L(EOS Platform Options,eos_platform_options).
+options:
+    config:
+      description: A list of route-map options
+      type: list
+      elements: dict
+      suboptions:
+        route_map:
+          description: Route map name.
+          type: str
+        entries:
+          description: Route Map entries.
+          type: list
+          elements: dict
+          suboptions:
+            statement:
+              description: statement name
+              type: str
+            source:
+              description: Rename/Copy configuration
+              type: dict
+              suboptions:
+                action:
+                  description: rename or copy configuration
+                  type: str
+                  choices: ["rename", "copy"]
+                source_map_name:
+                  description: Source route map name.
+                  type: str
+                overwrite:
+                  description: if True, overwrite existing config.
+                  type: bool
+            action:
+              description: Action for matching routes
+              type: str
+              choices: ["deny", "permit"]
+            sequence:
+              description: Index in the sequence.
+              type: int
+            continue_map:
```

Comment:
If this points to a sequence and not another route_map, can we change this to `continue_sequence`?